src/overlays/actors/ovl_En_Goroiwa/z_en_goroiwa.h

```diff
@@ -13,18 +13,18 @@ typedef struct EnGoroiwa {
     /* 0x014C */ EnGoroiwaActionFunc actionFunc;
     /* 0x0150 */ ColliderJntSph collider;
     /* 0x0170 */ ColliderJntSphElement colliderItems[1];
-    /* 0x01B0 */ Vec3f unk_1B0;
-    /* 0x01BC */ f32 unk_1BC;
-    /* 0x01C0 */ f32 unk_1C0;
+    /* 0x01B0 */ Vec3f prevUnitRollAxis;
+    /* 0x01BC */ f32 prevRollAngleDiff;
+    /* 0x01C0 */ f32 rollRotSpeed;
     /* 0x01C4 */ s16 waitTimer;
-    /* 0x01C6 */ s16 unk_1C6;
-    /* 0x01C8 */ s16 collisionTimer;
+    /* 0x01C6 */ s16 bounceCount;
+    /* 0x01C8 */ s16 collisionDisabledTimer;
     /* 0x01CA */ s16 endWaypoint;
     /* 0x01CC */ s16 currentWaypoint;
     /* 0x01CE */ s16 nextWaypoint;
     /* 0x01D0 */ s16 pathDirection;
     /* 0x01D2 */ u8 isInKokiri;
-    /* 0x01D3 */ u8 unk_1D3;
+    /* 0x01D3 */ u8 flags;
```

Comment:
We don't necessarily have to change it here but imo we should avoid using names of fields that are already in the base actor struct. In this case it might be better to have it as stateFlags or something else if it makes sense.